app/websockets/handlers.py

```
@@ -1,28 +1,54 @@
-
-        bot_response = ChatMessage(
-            message=f"Bot: You said '{user_msg.message}'", sender="bot", type="text"
-        )
 
-        await manager.send_message(bot_response.model_dump_json(), websocket)
+async def emit_event(manager, websocket, action, payload):
+    await manager.send_event(websocket, {"action": action, "payload": payload})
 
+async def handle_user_message(websocket: WebSocket, message: str, manager):
+    try:
+        data = json.loads(message)
```

Ruff (0.12.2)

Local variable `data` is assigned to but never used

Remove assignment to unused variable `data`

(F841)